artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml

```
@@ -0,0 +1,48 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: keda-external-scaler-azure-cosmos-db
+displayName: KEDA External Scaler for Azure Cosmos DB
+createdAt: 2021-11-11T00:00:00Z
+description: Scale applications based on changes to Azure Cosmos DB container data.
+license: Apache-2.0
+homeURL: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
+prerelease: true
+keywords:
+  - keda
+  - azure-cosmos-db
+  - autoscaling
+links:
+  - name: GitHub
+    url: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
```

Let's add our Helm registry to be complete
```suggestion
links:
  - name: Helm Registry
    url: https://github.com/kedacore/charts
  - name: GitHub
    url: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
```